onnx/test/shape_inference_test.py

```diff
@@ -174,6 +192,65 @@ def _compare_value_infos(
             )
 
 
+def single_op_all_version_tester(op_name: str):
+    """This is decorator to run one test case for all versions of an Op.
+
+    However, there is a creation issue for Reshape Operator under version 5, which
+    probably caused by the mismatch creation arguments in _make_graph.
+    Therefore, all test cases now only run when version > 5.
+    Issue reference: https://github.com/onnx/onnx/issues/5289.
+
+    Args:
+        op_name: The name of the Op, which should be in `defs.get_all_schemas`.
+
+    Returns:
+        The decorated function for unittest case.
+        For example, we can use:
+        ```
+        @single_op_tester("Transpose")
```

Comment:
Nit: name needs to be changed to match actual name "single_op_all_version_tester"

Reply:
Good catch...

Reply:
The decorator is also removed.